datasets/us/dos_terrorist_exclusion/crawler.py

```diff
@@ -0,0 +1,54 @@
+from zavod import Context, helpers as h
+import requests
+from typing import cast
+from lxml import html
+
+HEADERS = {
+    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.4.1 Safari/605.1.15",
+}
+
+
+def crawl_item(raw_name: str, context: Context):
+
+    entity = context.make("LegalEntity")
+
+    # If it does not have parenthesis, that it doesn't have aliases
+    if "(" not in raw_name:
+        name = raw_name
+        aliases = []
+
+    else:
+        name = h.remove_bracketed(raw_name)
+
+        res = context.lookup("aliases", raw_name)
+        if res:
+            aliases = cast("List[str]", res.names)
+        else:
+            context.log.warning(f"Could not find aliases for {raw_name}")
+            aliases = []
+
+    entity.id = context.make_id(name)
+    entity.add("name", name)
+
+    for alias in aliases:
+        entity.add("alias", alias)
+
+    entity.add("topics", "crime.terror")
+
+    sanction = h.make_sanction(context, entity)
```

Comment:
no need for a sanction entity if there isn't additional data about the ban like start/end date.

In this case the `program` could be `Section 411 of the USA PATRIOT ACT of 2001 (8 U.S.C. § 1182) Terrorist Exclusion List (TEL)` but that can go on the legal entity rather than the sanction entity since there's still no additional data for the sanction, making it overkill.